samples/apps/android/LwcTestApp/build.gradle

```diff
@@ -0,0 +1,29 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    
+    repositories {
+        google()
+        jcenter()
+        
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:3.6.3'
```

Comment:
Since we're in a new project, we should start with the latest version of the Gradle plugin (4.1.1).